mixpanel.js

```diff
@@ -4533,15 +4533,16 @@ Globals should be all caps
     // EXPORTS (for closure compiler)
 
     // Underscore Exports
-    _['toArray']         = _.toArray;
-    _['isObject']        = _.isObject;
-    _['JSONEncode']      = _.JSONEncode;
-    _['JSONDecode']      = _.JSONDecode;
-    _['isBlockedUA']     = _.isBlockedUA;
-    _['isEmptyObject']   = _.isEmptyObject;
-    _['info']            = _.info;
-    _['info']['device']  = _.info.device;
-    _['info']['browser'] = _.info.browser;
+    _['toArray']            = _.toArray;
+    _['isObject']           = _.isObject;
+    _['JSONEncode']         = _.JSONEncode;
+    _['JSONDecode']         = _.JSONDecode;
+    _['isBlockedUA']        = _.isBlockedUA;
+    _['isEmptyObject']      = _.isEmptyObject;
+    _['info']               = _.info;
+    _['info']['device']     = _.info.device;
+    _['info']['browser']    = _.info.browser;
+    _['info']['properties'] = _.info.properties;
```

Comment:
`people_properties` is just a minimal subset of `properties`, so I can't really think of an advantage to exposing it, at least not for the purposes expressed in https://github.com/mixpanel/mixpanel-js/issues/35